.github/workflows/build-linux-amd64.sh

```diff
@@ -2,31 +2,21 @@
 
 # Run this in Docker with the Zipline project mounted at /zipline.
 #
-# This script runs on CentOS 7 with devtoolset-7. We deliberately use an
-# old distro so our artifacts will run on similarly old distros. We need
-# to use Docker because GitHub actions doesn't offer virtual environments
-# for obsolete distros.
-#
-# We need the distribution's libc and libcxx to be old-enough so its outputs
-# work for our users. We also need a compiler that's new-enough that it has
-# `stdatomic.h`. Yuck.
+# We use Docker because GitHub actions doesn't offer virtual environments for old distros, and we
+# need to build on old distros to run on old distros.
 
 set -e
 set -x
 
 cd /zipline
-yum -y install centos-release-scl
-yum -y install devtoolset-7 devtoolset-7-toolchain
-. /opt/rh/devtoolset-7/enable
-yum -y install epel-release
-yum -y install cmake3
-yum -y install java-1.8.0-openjdk-devel
+
+dnf install gcc-c++ cmake3 java-11-openjdk-devel
```

Comment:
We should move this to the action YAML so that this script works on bare metal linux the same way the mac one works on macs. The Docker comment should also move there, I think.

Reply:
Done. I made a basic `Dockerfile`.